src/components/modals/FormStepSaveModal.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Display a modal to allow the user to save the form step in it's current state.
+ */
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {useHistory} from 'react-router-dom';
+import {FormattedMessage} from 'react-intl';
+import {put, post, destroy} from 'api';
+import {getBEMClassName} from 'utils';
+import Modal from 'components/modals/Modal';
+import ErrorMessage from 'components/ErrorMessage';
+import Body from 'components/Body';
+import Label from 'components/Label';
+import Input from 'components/Input';
+import HelpText from 'components/HelpText';
+import {Toolbar, ToolbarList} from 'components/Toolbar';
+import Button from 'components/Button';
+
```

Comment:
Please group and order your imports properly. External dependencies first, then our own stuff separated by a blank line and sort alphabetically within a group.